packages/next-ui/PageMeta/PageMeta.tsx

```diff
@@ -85,19 +86,19 @@ export function useCanonical(incoming?: Canonical) {
 
 export function PageMeta(props: PageMetaProps) {
   const { active } = usePageContext()
-  const { title, canonical: canonicalBare, metaDescription, metaRobots = ['all'] } = props
+  const { children, title, canonical: canonicalBare, metaDescription, metaRobots = ['all'] } = props
 
   const canonical = useCanonical(canonicalBare)
 
   if (!active) return null
   return (
-    <Head>
-      <title>{title.trim()}</title>
-      {metaDescription && (
-        <meta name='description' content={metaDescription.trim()} key='meta-description' />
-      )}
-      <meta name='robots' content={metaRobots.join(',')} key='meta-robots' />
-      {canonical && <link rel='canonical' href={canonical} key='canonical' />}
-    </Head>
+      <Head>
+        <title>{title.trim()}</title>
+        {metaDescription && (
+            <meta name='description' content={metaDescription.trim()} key='meta-description' />
+        )}
+        {canonical && <link rel='canonical' href={canonical} key='canonical' />}
```

Comment:
robots seems to be removed :)

Can you also make sure indenting is set to two spaces, seems to add indenting here?

Reply:
My bad, fixed :)